.env

```diff
@@ -1 +1,23 @@
 # This file is here for dynaconf to set this as root level directory
+
```

Comment:
There are no docs included guiding users on what to do for local configuration of vault.

This file should be a template, or this block dumped by vault_login script if not present.

The user can't modify this file without source control seeing it. The contents here should be in an example file, so that the user can have a `.env` file locally.

Reply:
Cool! This can be moved to example file, there is no harm though !